tests/org.palladiosimulator.dataflow.confidentiality.analysis.tests/src/org/palladiosimulator/dataflow/confidentiality/analysis/ListAppender.java

```diff
@@ -0,0 +1,37 @@
+package org.palladiosimulator.dataflow.confidentiality.analysis;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.apache.log4j.AppenderSkeleton;
+import org.apache.log4j.Level;
+import org.apache.log4j.spi.LoggingEvent;
+
+public class ListAppender extends AppenderSkeleton {
```

Comment:
Please add a short comment why this class is needed at all (debugging logging and so on)